BeerOverflowWindowsApp/DataModels/BarDataModel.cs

```diff
@@ -134,7 +134,7 @@ public void GetRatings()
         {
             foreach (var bar in this)
             {
-                bar.Ratings = BarFileReader.GetBarRatings(bar);
+                bar.Ratings = new DatabaseManager().GetBarRatings(bar);
```

Comment:
Is it necessary to create a new instance on every iteration?

Reply:
well if we make instances we can use interfaces and i think it will be useful when we will start thinking about running our app in offline mode and caching data in local files. Of course then we will have to implement something like a Factory method.

Reply:
I mean, can we make the DatabaseManager instance above the foreach?

Reply:
oh, yeah sure. Dont know why i missed that...